sass/assert/_utils.scss

```
@@ -127,7 +127,10 @@
   $not: (not not $assert);
   $list: ($assert != ());
   $string: ($assert != '');
-  $truthy: if(($not and $list and $string), true, false);
 
-  @return $truthy;
+  @if ($not and $list and $string) {
+    @return true;
+  } @else {
+    @return false;
+  }
```

No need for the else in this case

```suggestion
  @if ($not and $list and $string) {
    @return true;
  }
  
  @return false;
```